conf/metering_attr_fail.bess

```diff
@@ -0,0 +1,28 @@
+#This test case will test Metering logic and variables passing logic. with below configuration we shall get ~1.0mpps around on Gate 0 in visual graph .
+#Input for metering : .cir = 1000000 * 60, .cbs = 2048, .ebs = 2048 ; 
+#Input for testing whether module is passing corect value further   : foobar = 3
+#expected output : for metering User shall see ~1.0 Mpps outputting at gate 0 in graph, rest goes to Gate no 1 of Metering Module.
+#expected output : Split will choose output gate based on value of foobar  : for foobar=3, user shall see (all green packets) ~1.0 Mpps shall goto gate 3 of Split module.
+
+import socket
+def aton(val):
+    return socket.htonl(val)
+
+Metering::Qos(fields=[{'attr_name':'inport',
+              'num_bytes':4},{'attr_name':'outport', 'num_bytes':4}],
+              values=[{'attr_name':'foobar', 'num_bytes':4}])
+
```

Comment:
we can combine this file with offset file and test both in a single file

Reply:
move the combined file to bessctl/module_tests

Reply:
can also set the source packet size as per metering other params for getting certain metered output which tester can verify